src/node/node_object.hpp

```diff
@@ -26,15 +26,7 @@ namespace js {
 template<>
 inline v8::Local<v8::Value> node::Object::get_property(v8::Isolate* isolate, const v8::Local<v8::Object> &object, StringData key) {
     Nan::TryCatch trycatch;
-    v8::Local<v8::String> node_key;
-
-    // If we have just plain ASCII, we can skip the conversion from UTF-8
-    if (std::all_of(key.data(), key.data() + key.size(), [](char c) { return c <= 127; })) {
-        node_key = v8::String::NewExternal(isolate, new v8::ExternalOneByteStringResourceImpl(key.data(), key.size()));
```

Comment:
`v8::ExternalOneByteStringResourceImp` is sadly missing in latest v8. Should we implement it ourselves, is the optimization worth it?

Reply:
This was a pretty big speedup for the mssql loader, and not because the loader was doing anything unusual.

Reply:
Alright, I'll try to re-implement `v8::ExternalOneByteStringResourceImpl` - it's pretty simple so it should be okay.